ktor-client/ktor-client-java/jvm/src/io/ktor/client/engine/java/JavaHttpEngine.kt

```diff
@@ -103,7 +103,7 @@ public class JavaHttpEngine(override val config: JavaHttpConfig) : HttpClientEng
                 proxy(ProxySelector.of(address))
             }
             Proxy.Type.DIRECT -> proxy(HttpClient.Builder.NO_PROXY)
-            else -> throw IllegalStateException("Proxy of type $type is unsupported by Java HTTP engine.")
+            else -> throw IllegalStateException("Java HTTP engine does not currently support $type proxies.")
```

Comment:
Why do we need the word `currently`?

Reply:
Because at some point we'll support it, and it's to indicate that this isn't permanent.